python/modules/electronSF.py

```diff
@@ -1,13 +1,11 @@
-"""
-Compute muon SFs using correctionlib, and store in new branches.
-Load as: 
- eleSF = electronSF("POG/EGM/2016postVFP_UL/electron.json.gz")
- eleSF.addCorrection("NUM_TrackerMuons_DEN_genTracks", "2016postVFP", "sf")
- eleSF.addCorrection("NUM_MediumID_DEN_TrackerMuons", "2016postVFP", "sfdown", "sfsysdn")
- eleSF.addCorrection("NUM_MediumID_DEN_TrackerMuons", "2016postVFP", "sfup", "sfsysup")
-
-See example in test/example_electronSF.py for details.
-"""
+###
+# Compute muon SFs using correctionlib, and store in new branch.
```

Comment:
The comment here was using [docstring conventions](https://www.geeksforgeeks.org/python/python-docstrings/). Can you please revert it? Also, the reference to the example is useful.
BTW while looking at this  - "muon" at L2 comes from cut and paste... while editing you can maybe fix this.